### PR TITLE
test: [divider, split, tag] fix current test errors

### PR DESCRIPTION
**Pull request by gweesin**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

b75d906b87a6d682fb8f7b29c30f185d

Issue Number: N/A

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **Tests**
	- Updated test assertions for better alignment with the new styling conventions in various components.
	- Enhanced specificity of color definitions in tests for improved rendering consistency.
	- Modified event handler naming conventions in tests to reflect updated standards.
	- Improved type safety and corrected element indices in event handling tests for better accuracy.
	- Adjusted mouse movement distances and height expectations in tests for more precise UI behavior validation.
	- Simplified metadata structure in the grid API for improved clarity and usability. 
	- Removed placeholder configurations for better handling of empty data scenarios in components. 
	- Changed component status annotations from versioning to clearer stability markings in the menus configuration.

### CODE REVIEW

**Review by kagol — DISMISSED** [review]

**Comment by kagol** [comment]
@gweesin 以下命令可以单独执行某个组件的单元测试用例，这样验证起来会快一些
```
pnpm test:unit3 button
```

**Comment by gweesin** [comment]
> @gweesin 以下命令可以单独执行某个组件的单元测试用例，这样验证起来会快一些
> 
> ```
> pnpm test:unit3 button
> ```

ok了解了！我之前单独跑的时候直接改vite.config.ts配置文件的单元测试扫描路径来做调试，多此一举了🤣

**Comment by kagol** [comment]
@gweesin split/tag组件的e2e测试用例报错

File: examples/sites/demos/pc/app/split/movestart-event.spec.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ test('movestart 事件', async ({ page }) => {
...
-  const { x, y } = await centerBtn.boundingBox()
+  const { x, y } = (await centerBtn.boundingBox())!
   // 鼠标按下
   await centerBtn.hover()
   await page.mouse.down()
   await page.mouse.move(20 + x, y)
   await page.mouse.up()
   //   向右移动之后判断是否出现提示
-  const moveDiv = page.getByText('拖拽开始').nth(1)
+  const moveDiv = page.getByText('拖拽开始').nth(0)
```
Comment (zzcr) [inline examples/sites/demos/pc/app/split/movestart-event.spec.ts]: 你好，e2e测试用例我们内部已经修改完了哈，这个mr可以把e2e的修改去掉，保留单元测试用例的修改就好了哈，改完我给你合入下，感谢您的参与

**Comment by zzcr** [comment]
你好，代码有冲突，修改下，我来合入哈

**Comment by zzcr** [comment]
这么mr貌似有点问题740个文件改动，是不是分支污染了，可以重新拉个分支提交

**Comment by gweesin** [comment]
> 这么mr貌似有点问题740个文件改动，是不是分支污染了，可以重新拉个分支提交

确实奇怪，我based分支压根没这些调整，但却要我处理这些conflict，我看看重新拉个分支

**Comment by zzcr** [comment]
已合入新拉的分支，这个mr我就关闭了哈

**Pull request closed without merging**